com.civicrm.hrjobroles/js/src/job-roles/controllers/job-role-panel.html

```diff
@@ -362,7 +362,9 @@
                           <ui-select-choices
                             prevent-animations
                             class="ui-select-choices"
-                            repeat="funder.value.value as (key,funder) in jobroles.FundersData | filter: { value: { title: $select.search }}">
+                            repeat="funder.value.value as (key,funder) in jobroles.fundersData | filter: { value: { title: $select.search }}"
+                            refresh="jobroles.getOptionGroupTypeValues('hrjc_funder')"
+                            refresh-delay="500">
```

Comment:
Same as above.

Reply:
This was omitted in the initial implementation while refactoring the ui-select.